wpigui/src/main/native/include/portable-file-dialogs.h

```diff
@@ -146,7 +147,7 @@ class dialog : protected settings
     std::shared_ptr<executor> m_async;
 };
 
-class file_dialog : public dialog
+class file_dialog : public internal::dialog
```

Comment:
This change shouldn't be required?

Reply:
It isn't required, but I made the change for it to be consistent with the other dialog classes being explicit about the namespace